File: Mep.Api/ViewModels/ReferralEdit.cs
using System;

namespace Mep.Api.ViewModels
{
  public class ReferralEdit
  {
    public DateTimeOffset CreatedAt { get; set; }
    public int Id { get; set; }
    public string LeadAmhpUserDisplayName {get; set;}
    public int LeadAmhpUserId { get; set; }    
    public string PatientAlternativeIdentifier { get; set; }
    public int? PatientCcgId { get; set; }
    public string PatientCcgName {get; set;}
    public string PatientGpPracticeNameAndPostcode {get; set;}
    public int? PatientGpPracticeId { get; set; }
    public long? PatientNhsNumber { get; set; }
    public string PatientResidentialPostcode { get; set; }
    public string Status { get; set; }
  }
}

File: Mep.Ui/src/app/interfaces/referralEdit.ts
export interface ReferralEdit {
  createdAt: Date;
  id: number;
  isPlannedExamination: boolean;
  leadAmhpUserId: number;
  leadAmhpUserDisplayName: string;
  patientAlternativeIdentifier: string;
  patientCcgId: number;
  patientCcgName: number;
  patientGpPracticeId: number;
  patientGpNameAndPostcode: number;
  patientNhsNumber: number;
  patientResidentialPostcode: string;
  status: string;
}
